### ISSUE TITLE
zero-1 combined with fusedlamb  works well

### ISSUE THREAD

**Issue by gongjingcs** [microsoft/DeepSpeed#628]
In Bert pre-training, I combined the lamb optimizer with zero-stage1 and found that the loss decline is normal, indicating that the zero-1 and fusedlamb optimizers are compatible, but the zero code says that lamb optimizers are not supported
![image](https://user-images.githubusercontent.com/15030497/103507705-bb587a00-4e9a-11eb-9433-8a2ab59e05c2.png)

In addition, from the lamb formula, zero-1 can also partition the optimizer state, which is the same as the partition for the adam optimizer.
